packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -229,6 +251,7 @@ class ConnectionStateHandler implements IConnectionStateHandler {
 
     constructor(
         private readonly handler: IConnectionStateHandlerInputs,
+        private readonly readClientsWaitForJoinSignal: boolean,
         private _clientId?: string,
```

Comment:
Unrelated to this change, but I got confused about _clientId.  Maybe name the ctor parameter `clientIdFromPausedSession` or something different, since the parameter has that specific meaning, and in the regular flow it is undefined (but `this._clientId` semantics are the same for regular sessions and deserialized sessions alike)